packages/tree-view-for-tests/src/__tests__/tree.test.ts

```diff
@@ -6,33 +6,18 @@ test('it can serialise a folder', () => {
   // so can't include any .test.ts files that actually use this.
   // I picked one of our packages instead.
 
-  expect(tree(path.join(__dirname, '../../../create-modular-react-app')))
+  expect(tree(path.join(__dirname, '../../../modular-views.macro')))
```

Comment:
CMRA uses tree in its tests so it causes too many snapshot updates to this test tile